Klarna.Rest/SampleProjects/KlarnaCheckoutWebApp/README.md

```diff
@@ -0,0 +1,27 @@
+ASP.NET [MVC Web App](https://dotnet.microsoft.com/apps/aspnet/mvc) Sample Project
+=============================================
+
+The ASP.NET MVC sample project demonstrates how to work with the Klarna Checkout and Klarna Payments.
+
+## Run Project
+
+Use IIS server or Visual Studio to run the Solution.  
+
+**Note**: Follow the [Microsoft guideline](https://docs.microsoft.com/en-us/visualstudio/ide/quickstart-aspnet-core?view=vs-2019)
```

Comment:
We could also use the CLI to run the application right?

Maybe create another ticket for the documentation change.

Reply:
It would be probably nice to have, but the problem - there are too many ways how to run the app. I don't want to have a descriptions for all of them. Actually, if you are a .net developer - you should now how to run the solution: it can be console, IDE, etc.

Talking about the console - it has a number of cases, like .NET SDK version and installation folder, mono version, etc. So it would be hard to cover all the cases.

Reply:
It would be still nice to have this. Its specially helpful for new developers wanting to contribute and understand how this project is working. Also, Visual Studio and CLI are probably the most popular ways to run the app, next to docker. I have never seen anyone running a .NET Core ASP.NET web app in mono. Anyway, I am fine with not doing it in this PR. I can create tickets - actually its perfect tickets for new contributors :)

I mean markdown supports good ways to not clutter the markdown like using something like this:

```md
<p><details>
  <summary><b>Using Visual Studio</b></summary>
  
  ## Markdown works Lorem ipsum
</details></p>

<p><details>
  <summary><b>Using the CLI</b></summary>
  
  ## Markdown works Lorem ipsum
</details></p>
```

**Example:**

<p><details>
  <summary><b>Using Visual Studio</b></summary>
  
  ## Markdown works Lorem ipsum
</details></p>

<p><details>
  <summary><b>Using the CLI</b></summary>
  
  ## Markdown works Lorem ipsum
</details></p>